NitroxClient/GameLogic/Spawning/WorldEntities/CrashEntitySpawner.cs

```diff
@@ -15,7 +15,7 @@ public IEnumerator SpawnAsync(WorldEntity entity, Optional<GameObject> parent, E
         {
             if (parent.HasValue)
             {
-                CrashHome crashHome = parent.Value.GetComponentInParent<CrashHome>();
+                CrashHome crashHome = parent.Value.GetComponent<CrashHome>();
```

Comment:
Doesn't `GetComponentInParent` check the current game object first?  Surprised this would cause issues.

Reply:
I had checked what parent.Value was via logging, It was CrashHome (clone)... So I tried changing it to GetComponent and that ended up fixing it

Reply:
Yea i read that too...